### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by AS3jupiter** [3kh0/ext-remover#381] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I've been trying to use the other 106 and up method javascript:localStorage.cluster="UNKNOWN_SCHOOL,"+(confirm("ok = Disable\ncancel = Enable ")?99999999999999:0),opener.chrome.extension.getBackgroundPage().location.reload() but when I got to the website all it says "Your file couldn’t be accessed It may have been moved, edited, or deleted. ERR_FILE_NOT_FOUND" and I don't know what to do.

when I use the other method GoGuardian turns back on after some time so I wanted to try the other one and it takes me to that page that I said previously and I don't know if it's supposed to do that

**Comment by rwg100** [comment, marked as answer]
goguardian = skill issue!!!!

**Reply by rwg100** [reply]
dyknow = also hate!!!

**Comment by Aka-but-nice** [comment]
> I've been trying to use the other 106 and up method javascript:localStorage.cluster="UNKNOWN_SCHOOL,"+(confirm("ok = Disable\ncancel = Enable ")?99999999999999:0),opener.chrome.extension.getBackgroundPage().location.reload() but when I got to the website all it says "Your file couldn’t be accessed It may have been moved, edited, or deleted. ERR_FILE_NOT_FOUND" and I don't know what to do.
> 
> when I use the other method GoGuardian turns back on after some time so I wanted to try the other one and it takes me to that page that I said previously and I don't know if it's supposed to do that

You have goguardian right

**Reply by AS3jupiter** [reply]
Ye

**Reply by Aka-but-nice** [reply]
Alr your screwed

**Reply by AS3jupiter** [reply]
what do you mean

**Reply by Aka-but-nice** [reply]
They updated goguardian it doesn't work anymore

**Reply by AS3jupiter** [reply]
damnnn ight

**Reply by Aka-but-nice** [reply]
Yeah
